build/ci/nightly/Jenkinsfile

```diff
@@ -31,6 +31,7 @@ pipeline {
                 sh """
                     export VERSION=\$(cat $WORKSPACE/operators/VERSION)-\$(date +%F)-\$(git rev-parse --short --verify HEAD)
                     export OPERATOR_IMAGE=${REGISTRY}/${REPOSITORY}/${IMG_NAME}:\$VERSION
+                    export LATEST_RELEASED_IMG=docker.elastic.co/${REPOSITORY}/${IMG_NAME}:\$VERSION
```

Comment:
Is OPERATOR_IMAGE where we publish and LATEST_RELEASED_IMG from where we pull?

Reply:
Not exactly. `OPERATOR_IMAGE` is as you described, but `LATEST_RELEASED_IMG` is needed for https://github.com/elastic/cloud-on-k8s/blob/master/operators/Makefile#L179

Reply:
Ah, yes, that's what I was thinking (pull as per k8s pulls our image).